### DISCUSSION TITLE
Changing  channel in configuration.yaml impact to currently paired devices

### DISCUSSION THREAD

**Discussion by tmichaeltx** [Koenkk/zigbee2mqtt#6316] [Devices]
I have a device that is limited to which channels it can use. My very stable setup is using a channel that the device doesn't support. If I change the channel in the configuration.yaml, will this affect the device that are already paired to it?

**Comment by tmichaeltx** [comment]
Any thoughts or knowledge from anyone?

**Reply by Koenkk** [reply]
Check https://www.zigbee2mqtt.io/information/FAQ.html#requires-repairing

**Reply by tmichaeltx** [reply]
Thanks for the pointer to answer my question.
